# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/tests/non262/fields/access.js
# Commit: 1b8c42577e24
# Full Hash: 1b8c42577e2410dbbe050bb30b0ccced9d7c3771
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-20 10:52:37
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r 28370bf14b97 -r 1b8c42577e24 js/src/tests/non262/fields/access.js
--- a/js/src/tests/non262/fields/access.js	Tue Mar 19 22:02:49 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,19 +0,0 @@
-// * * * THIS TEST IS DISABLED - Fields are not fully implemented yet
-
-class C {
-    x = 5;
-}
-
-c = new C();
-
-reportCompare(c.x, undefined); // TODO
-//reportCompare(c.x, 5);
-
-class D {
-    #y = 5;
-}
-
-d = new D();
-
-reportCompare(d.#y, undefined); // TODO
-//reportCompare(d.#y, 5);